# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../madwifi/macbook.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- madwifi-hal-0.9.30.10-r2230-20070328/ath/if_ath_pci.c.orig	2007-03-28 20:05:06.298719627 +0200
+++ madwifi-hal-0.9.30.10-r2230-20070328/ath/if_ath_pci.c	2007-03-28 20:05:52.485351653 +0200
@@ -103,6 +103,7 @@
 	{ 0x168c, 0x001c, PCI_ANY_ID, PCI_ANY_ID }, /* PCI Express 5424 */
 	{ 0x168c, 0x001d, PCI_ANY_ID, PCI_ANY_ID }, /* PCI Express ???  */
 	{ 0x168c, 0x9013, PCI_ANY_ID, PCI_ANY_ID }, /* sonicwall */
+	{ 0x168c, 0x0024, PCI_ANY_ID, PCI_ANY_ID },
 	{ 0 }
 };
 
